EncoreTickets.ConsoleTester/Program.cs

```diff
@@ -14,24 +14,41 @@ class Program
     {
         static void Main(string[] args)
         {
-            ApiContext context = new ApiContext(Environments.Sandbox);
-            List<string> productIds = new List<string>();
-            context.affiliate = "encoretickets";
-            // uscontext.useBroadway = true;
+            var context = new ApiContext(Environments.Sandbox, "admin", "password") {affiliate = "encoretickets"};
+            // context.useBroadway = true;
+            var productIds = new List<string>();
+
+            Console.WriteLine();
+            Console.WriteLine(" ========================================================== ");
+            Console.WriteLine(" Test: Get JWT token for the venue service");
+            Console.WriteLine(" ========================================================== ");
+            var vsAuthApi = new VenueAuthenticationServiceApi(context);
+            context = vsAuthApi.Authenticate();
+
+            Console.WriteLine($"username: {context.userName}");
```

Comment:
I know this is part of debugging, but it shouldn't be merged - or will you remove before the last PR?

Reply:
If you do not need this console project, I will delete it all in the last PR, ok? Currently, it is useful

Reply:
ok, let's see in the last PR - usually speaking, we should log using logging channels and to files and avoid logging passwords